### DISCUSSION TITLE
Informações sobre controle de contas de redes sociais pela Apyb

### DISCUSSION THREAD

**Discussion by thaty-maru** [apyb/comunidade#165] [Pergunta]
Boa tarde, time 

Meu nome é Thatianne, faço parte da organização da Python Norte 2025, evento apoiado pela Apyb.
Foi iniciada no grupo de telegram PyNorte uma discussão sobre o domínio das redes sociais pela Apyb, pois atualmente as redes sociais referentes ao evento (@pynorteam) estão em posse de outras pessoas, e todo ano temos que solicitar autorização para o seu uso. Para este ano solicitamos o acesso total das redes sociais para facilitar nossa utilização, e transferir o controle para a Apyb, e chegou-se na seguinte pergunta:

> Pelo o que eu entendi, organização da conferência de 2025, via ou com auxílio do Mateus Santos atualmente está solicitando que seja passado o acesso total e irrestrito e que essas credenciais sejam salvas em alguma conta de bitwarden gerenciada por terceiros (suponho que o terceiro seja a APyB).
> Caso o terceiro seja mesmo a APyB, qual é a política de segurança da informação e etc a APyB como entidade jurídica irá fornecer e assim resguardar qualquer mal uso intencional ou não dos acessos a essas contas, e como será gerenciado a delegação de permissões?

Gostaria de solicitar essa informação para fins de esclarecimento para todos os envolvidos.

Obrigada.

**Comment by jailsonpj** [comment]
@naanadr @rougeth (te marcando aqui pra ajudar tbm 😅)

**Reply by naanadr** [reply]
Olá @jailsonpj a APyB não gerencia os usuários e senhas das redes sociais dos eventos regionais, apenas fazemos a criação do e-mail no Google e vocês definem a senha. 

O que estamos recomendando, é os eventos regionais solicitarem no 1password acesso a contra Premium, pelo apoio a projetos open source que eles oferecem.

Qualquer dúvida em relação a isso é só perguntar.

🔗 Mais info sobre o programa: https://github.com/1Password/for-open-source
🔗 Solicitação do 1password para a PyNe: https://github.com/apyb/comunidade/discussions/130

**Reply by jailsonpj** [reply]
Obrigado @naanadr
